pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -607,6 +607,8 @@ type ClusterAutoscaler struct {
 	// IgnoreDaemonsetsUtilization allows CA to ignore DaemonSet pods when calculating resource utilization for scaling down.
 	//+optional
 	IgnoreDaemonsetsUtilization *bool `json:"ignoreDaemonsetsUtilization,omitempty" protobuf:"varint,13,opt,name=ignoreDaemonsetsUtilization"`
+	// Verbosity allows CA to modify it's log level
```

Comment:
```suggestion
	// Verbosity allows CA to modify it's log level.
```
And for the internal API as well.